pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +536,30 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func Test_getSecurityGroupName(t *testing.T) {
+
+	service := &corev1.Service{ObjectMeta: v1.ObjectMeta{UID: "12345", Namespace: "security-group-namespace", Name: "security-group-name"}}
+	expected := "lb-sg-12345-security-group-namespace-security-group-name"
+	t.Run("regular test security group name and length", func(t *testing.T) {
+		if got := getSecurityGroupName(service); got != expected {
+			t.Errorf("getSecurityGroupName() = %s expected = %s", got, expected)
+		}
+	})
```

Comment:
as @dulek suggested, let's try to stick to DRY(don't repeat yourself) principle rather than initialising the run block with mock values every time.